src/_pytest/capture.py

```diff
@@ -681,7 +681,8 @@ def resume(self):
         setattr(sys, self.name, self.tmpfile)
         self._state = "resumed"
 
-    def writeorg(self, data):
+    def writeorg(self, data: str) -> None:
```

Comment:
This needs to be `bytes`. It seems like the `data = data....` confused mypy so it didn't catch it.